.ddev/commands/web/orchestrate.d/33_woocommerce_config.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
This are starting settings that we usually need and can be easily changed on a new build for different tests

Reply:
In PayPal Payments it is executed before running tests every time, to avoid confusion if modifying these settings for manual tests, and to avoid env rebuild (destruction) when adding/changing this config.

Also it seems to be much faster without wp cli.

https://github.com/woocommerce/woocommerce-paypal-payments/blob/bf4a94099fa372fefa7c5b6659320e8fba56ad03/package.json#L31

https://github.com/woocommerce/woocommerce-paypal-payments/blob/trunk/tests/e2e/PHPUnit/setup.php